app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
```

Comment:
Add type hints for the friends parameter and the return type.
`
from typing import List, Dict
def go_to_cafe(friends: List[Dict], cafe: Cafe) -> str:
`